Signal/src/ViewControllers/HomeView/ConversationSearchViewController.swift

```diff
@@ -36,6 +36,36 @@ class ConversationSearchViewController: UITableViewController {
         tableView.register(MessageSearchResultCell.self, forCellReuseIdentifier: MessageSearchResultCell.reuseIdentifier)
     }
 
+    // MARK: UITableViewDelegate
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: false)
+
+        guard let searchSection = SearchSection(rawValue: indexPath.section) else {
+            owsFail("\(logTag) unknown section selected.")
+            return
+        }
+
+        var sectionResults: [SearchResult]
+        switch searchSection {
+        case .conversations:
+            sectionResults = searchResultSet.conversations
+        case .contacts:
+            sectionResults = searchResultSet.contacts
+        case .messages:
+            sectionResults = searchResultSet.messages
```

Comment:
Just making sure you noticed there is a separate card to go to the precise message that was matched by the search. This is good stand-in functionality for now though. 👍

Reply:
Yup.